llm-server/utils/makechain.ts

```diff
@@ -36,6 +36,7 @@ If the question is not related to the context, politely respond that you are tun
 {context}
 
 Question: {question}
+(Answer in the same language as the {question})
```

Comment:
To save the number of tokens, can we do this instead?

```
Helpful answer in same language as the question and in markdown:
```

Reply:
good👍🏻

Reply:
resolve